app/assets/javascripts/discourse/app/components/composer-body.js

```diff
@@ -199,6 +199,34 @@ export default Component.extend(KeyEnterEscape, {
     );
   },
 
+  updateHeightOnViewportResize() {
+    window.addEventListener(
+      "resize",
+      () => {
+        const composerState = this.get("composer.composeState");
+        if (composerState !== Composer.OPEN) {
+          return;
+        }
+
+        const composerElementHeight =
+          document.querySelector("#reply-control").offsetHeight;
+        const currentHeight = parseInt(this.composer.composerHeight, 10);
+
+        if (
+          composerElementHeight < currentHeight &&
+          composerElementHeight > 255
```

Comment:
Elsewhere on this page, we do this: 

https://github.com/discourse/discourse/blob/main/app/assets/javascripts/discourse/app/controllers/composer.js#L486-L490 

I guess we can make sure height changed before storing it... but not sure why we are checking it's more than 255px tall.

Reply:
Yes, not sure why I had the check for `255px` and the height change check doesn't seem necessary either. Also after commit https://github.com/discourse/discourse/commit/9d9c48fcb867b6ffd88e84f670a3e084a8d9bea7 the check causes issues because it starts the value composerHeight value is `var(--new-topic-composer-height, 400px);
}` which is difficult to parse.

In any case, I've removed the check.